cluster/config-defaults.yaml

```diff
@@ -650,6 +650,7 @@ blocked_availability_zone: ""
 
 # etcd cluster
 etcd_stack_name: "etcd-cluster-etcd"
+etcd_dns_record_prefixes: "etcd-server.{{.Cluster.Region}}"
```

Comment:
nit: why do we have a plural name? `prefixes` instead of `prefix`? Just curious.

Reply:
It's because we allow multiple for migration. Let me add a comment in a follow up to make this more clear